### DISCUSSION TITLE
none of the folders get generated

### DISCUSSION THREAD

**Discussion by alexgorcas** [d8ahazard/sd_dreambooth_extension#1022] [Q&A]
when I press create it doesn't generate the needed folders to start and when I add the folders manualy to start Dreambooth so many files are missing, is there a step I missed?
![uhhh](https://user-images.githubusercontent.com/35354690/222991393-c4414d04-3d1c-4748-b8d3-654e34d2370e.PNG)

**Comment by Arilziem** [comment]
Apparently the model is not being created/extracted properly. It's not really about the folders but rather about the models that should be extracted into those folders.

Are there any error messages when you click "create"?
